src/main/java/com/mollin/yapi/YeelightDevice.java

```diff
@@ -310,4 +324,9 @@ public void setName(String name) throws YeelightResultErrorException, YeelightSo
         YeelightCommand command = new YeelightCommand("set_name", name);
         this.sendCommand(command);
     }
+
+    public YeelightDeviceMeta getMeta() {
```

Comment:
If we keep "meta", getter must return an Optional and not a default meta with empty attributes